packages/darklang/languageTools/parser/cliScript.dark

```diff
@@ -146,10 +158,12 @@ module Darklang =
           (pm: ProgramTypes.PackageManager.PackageManager)
           (filename: String)
```

Comment:
Why do we have both `scriptName` and `fileName`? is file name the full path?

Reply:
Hm, I'm not sure. It doesn't seem like `fileName` is even used?

Reply:
yes, it isn't used

Reply:
probably cool to remove it, then, unless I'm missing something